react-native-libraries.json

```
@@ -14379,5 +14379,22 @@
     ],
     "ios": true,
     "android": true
+  },
+  {
+  "githubUrl": "https://github.com/dpakakpo4/rn-use-google-places-autocomplete",
+  "npmPkg": "https://www.npmjs.com/package/rn-use-google-places-autocomplete",
+  "ios": true,
+  "android": true,
+  "web": false,
+  "windows": false,
+  "macos": false,
+  "tvos": false,
+  "visionos": false,
+  "expoGo": true,
+  "fireos": false,
+  "unmaintained": false,
+  "dev": false,
+  "template": false,
+  "newArchitecture": true
```

Hey @dpakakpo4, thanks for adding new entry to the directory! 👍 

CI fails since `npmPkg` field should contain package name, not URL. It's mostly used for scoped packages, and in case of package below it can skipped fully, since GitHub repository name matches the package. Also, let's remove `false` fields for redability:

```suggestion
  },
  {
    "githubUrl": "https://github.com/dpakakpo4/rn-use-google-places-autocomplete",
    "ios": true,
    "android": true,
    "expoGo": true,
    "newArchitecture": true
```

---

Hi @Simek 😎
Thank you for reviewing my PR. I'll remove the false fields and replace the URL with the package name in the npmPkg field.

---

Hi @Simek
It's done. Have a nice day.